### PR TITLE
Fix the measure_coverage.sh script

### PR DESCRIPTION
**Pull request by rtakacs**
Removed the old `--no-check-test` option from the build flags.

### CODE REVIEW

**Comment by robertsipka** [comment]
Could you please also remove from other places (e.g. Build-Script.md)

**Comment by rtakacs** [comment]
@robertsipka Nice catch! Of course :)

**Comment by rtakacs** [comment]
@robertsipka I've updated the files to use the new `run-test` option instead of the old `no-check-test` one.

File: docs/build/Build-Script.md [hunk 1]
```diff
@@ -235,6 +227,14 @@ To build for nuttx os, nuttx home directory must be given.
 ./tools/build.py --target-os=nuttx --target-arch=arm --target-board=stm32f4dis --nuttx-home="..."
 ```
 
+--
+#### `--run-test`
+With given this option, unit test checking will be performed.
```
Comment (glistening) [inline docs/build/Build-Script.md]: @rtakacs Although it is not a part of this PR, what about "run unit tests after build"? I would remove all occurrences of `With given this option` in explanation for each option. Without this phrases, it is still clear the each sentence says about each option.

**Pull request merged**